peps/pep-0352.rst

```diff
@@ -276,9 +273,6 @@ References
 .. [#hierarchy-good] python-dev Summary for 2004-08-01 through 2004-08-15
    http://www.python.org/dev/summary/2004-08-01_2004-08-15.html#an-exception-is-an-exception-unless-it-doesn-t-inherit-from-exception
 
-.. [#SF_1104669] SF patch #1104669 (new-style exceptions)
-   https://bugs.python.org/issue1104669
```

Comment:
I would keep a link to that issue, now https://github.com/python/cpython/issues/41459.